e2e/session-launcher.test.ts

```diff
@@ -1,23 +1,187 @@
 import {
-  createSession,
   deleteSession,
   loginAsAdmin,
   loginAsUser,
   navigateTo,
 } from './test-util';
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
+
+const getMenuItem = (page: Page, menuName: string) => {
+  return page.getByRole('link', { name: menuName });
+};
+
+const getCardItemByCardTitle = (page: Page, title: string) => {
+  return page.locator(`.ant-card:has-text("${title}")`);
+};
```

Comment:
The code to get the selector based on the menu item click or the title of the ant-card is likely to be used a lot in other test code. In this case, it seems like a good idea to write it as a util function to increase reusability.